src/tools/tool_cmdline.rs

```diff
@@ -262,18 +271,21 @@ async fn execute_background_command(
                 ));
             }
         }
-        tracing::info!("SERVICE START workdir {}:\n{:?}", cmdline_workdir, command_str);
+        info!("SERVICE START workdir {}:\n{:?}", cmdline_workdir, command_str);
         actions_log.push_str(&format!("Starting service with the following command line:\n{}\n", command_str));
 
         let mut command = create_command_from_string(&command_str, cmdline_workdir).await?;
-        let mut process = command
-            .stdout(Stdio::piped())
-            .stderr(Stdio::piped())
-            .spawn()
-            .map_err(|e| format!("failed to create process: {e}"))?;
-
-        let mut stdout_reader = BufReader::new(process.stdout.take().ok_or("Failed to open stdout")?);
-        let mut stderr_reader = BufReader::new(process.stderr.take().ok_or("Failed to open stderr")?);
+        command.stdout(Stdio::piped());
+        command.stderr(Stdio::piped());
+        let mut command_wrap = TokioCommandWrap::from(command);
+        #[cfg(unix)]
+        command_wrap.wrap(ProcessGroup::leader());
+        #[cfg(windows)]
+        command_wrap.wrap(JobObject);
```

Comment:
From where this JobObject comes from?, I don't see any imports, and I don't see any external library installed that provides it.

Reply:
it comes from process_wrap package
use process_wrap::tokio::*;

I didn't test it yet on win, but it should work as in linux (tested and work well)

Reply:
Ahh I see, I did'n know that syntax in rust, any my IDE was not giving me any hint (probably because I'm not in Windows), that's good